src/docs/Foundations/Typography/BodyStyles/index.mdx

```diff
@@ -6,22 +6,25 @@ status:
   scss: 'wip'
 ---
 
-> The body patterns provide basic typographic styles for readable multilines text or paragraphs like a product description or an article.
+> A body style in typography is the main text part of a pattern or a page. It must be identity and provide basic typographic styles for readable multilines texts or paragraphs.
 
-Body styles are patterns meant to help you use consistent typographic hierarchy with the context and across responsive viewports. It's mainly reserved for users wishing to focus on tasks with added value and user experience. Text classes are families of several font-sizes and help create a hierarchy of contents.
+## Our recommendation
 
-#### Body styles are defined with three factors :
+For readability and accessibility purposes we recommend you to use our regular value for body styles called **body medium**, corresponding to the following values:
 
-- Font-size _(that may vary depending on the viewport)_
-- Line-height _(that are optimised for readability and accessibility)_
-- Font-weight _(regular or semi-bold)_
+- Font-size: 16px
+- Font-weight: Regular
+- Line-height: 1.35 (= M)
 
-#### And comes with two variations for each classe :
+|                                         |                                      |
+| --------------------------------------- | ------------------------------------ |
+| <Preview path="typographyBodyMedium" /> | **Body Medium** <br /> 16px (1mu)    |
```

Comment:
Remove previews from tables. (for responsive purposes, and it break when using a different viewport in the toolbar of the previews)

Reply:
add the `nude` attribute (always in usage pages)